keras/ops/core.py

```diff
@@ -204,6 +212,46 @@ def slice_update(inputs, start_indices, updates):
     return backend.core.slice_update(inputs, start_indices, updates)
 
 
+@keras_export("keras.ops.scan")
+def scan(f, init, xs, length=None, reverse=False):
```

Comment:
You can just add `unroll` everywhere for consistency, even if it's not used by some backends.